examples/tfengine/full.yaml

```diff
@@ -112,11 +112,17 @@ templates:
         IAM_MEMBERS:
         - ROLE: "roles/storage.objectViewer"
           MEMBER: "group:example-readers@example.com"
-- name: "project-example-apps"
+- name: "project-example-prod-apps"
   recipe_path: "{{$BASE}}/folder/project.yaml"
   output_ref: "folder-prod-team1.dir"
   data:
     PROJECT_ID: "example-prod-apps"
+    APIS:
+    - "container.googleapis.com"
+    SHARED_VPC:
+      HOST: "example-prod-networks"
+      SUBNETS:
+      - "projects/example-prod-networks/regions/us-central1/subnetworks/example-subnet"
```

Comment:
Is a user expected to figure this string out? Would be great if this could just be "example-subnet", and the rest figured out.

Reply:
How does it look now?